integration-templates/google-drive/types.ts

```diff
@@ -0,0 +1,52 @@
+export interface GoogleDriveFileResponse {
+    id: string;
+    name: string;
+    mimeType: string;
+    webViewLink: string;
+}
+
+export interface Metadata {
+    files?: string[];
+    folders?: string[];
+}
+
+interface MimeTypeMapping {
+    mimeType: string;
+    responseType: 'text' | 'stream';
+}
+
+export const mimeTypeMapping: Record<string, MimeTypeMapping> = {
+    // Documents
+    'application/vnd.google-apps.document': { mimeType: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', responseType: 'text' },
```

Comment:
where does this mapping and the one below come from?

Reply:
- `application/vnd.google-apps.document`: This is a MIME type used by Google Drive to represent Google Docs documents.
- `mimeType`: `application/vnd.openxmlformats-officedocument.wordprocessingml.document` specifies the desired MIME type for the converted file, which is the standard MIME type for Microsoft Word documents in the Office Open XML format.